scripts/config.py

```diff
@@ -179,22 +179,12 @@ def gather_user_data_s3(s3_url):
 
     config = json.loads(data)
 
-    # if it has 'local_cidr', then use that value for l2tp_cidr
```

Comment:
This is going to invalidate config files that our customers have stored in s3. We'll need to come up with a way to communicate that this build isn't backwards compatible.

Reply:
I think we can just revert this, the issue here is that our example json content is outdated.